UnitTests/Onboarding/ContextualOnboarding/BrowserTabViewControllerOnboardingTests.swift

```diff
@@ -17,7 +17,7 @@
 //
 
 import XCTest
-import SwiftUI
+import struct SwiftUI.AnyView
```

Comment:
This is harmful in Xcode 15, but otherwise required on Xcode 16, because the new SwiftUI.Tab clashes with our Tab class 😬

Reply:
Thanks, I have this on a branch already but decided to hold off until I had some other Xcode 16 fixes ready - glad to have this fix in place now. 🙌